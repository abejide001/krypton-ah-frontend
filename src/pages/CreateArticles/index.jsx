import React from 'react';
import CreateArticleTab from '../../components/CreateArticleTab';

function CreateArticles() {
  return <CreateArticleTab />;
}

export default CreateArticles;
