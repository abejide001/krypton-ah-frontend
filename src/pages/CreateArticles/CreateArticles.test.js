import React from 'react';
import { shallow } from 'enzyme';
import CreateArticles from '.';

describe('Create Articles', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<CreateArticles />);
    expect(wrapper).toMatchSnapshot();
  });
});
