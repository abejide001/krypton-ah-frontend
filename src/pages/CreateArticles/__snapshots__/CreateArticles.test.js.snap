// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Create Articles should match snapshot 1`] = `<CreateArticleTab />`;
